### PR TITLE
✅  Add a11y test for Input

### LINKED ISSUE #1959: Add a11y test for Input

### PR DESCRIPTION
**Pull request by laliq**
Resolves #1959

### CODE REVIEW

**Comment by laliq** [comment]
Here is how [axe](https://dequeuniversity.com/rules/axe/4.2/label?application=axeAPI) describes the issue and solution.

**Comment by mimarz** [comment]
> Here is how [axe](https://dequeuniversity.com/rules/axe/4.2/label?application=axeAPI) describes the issue and solution.

Jepp, its a good explanation for the a11y rules for `input` fields :) 

Since we are basically just wrapping an `input` field with our `Input` component I suggest making two a11y test that cover the use-cases in which we used this component. One for `aria-label` and the other using a `label`, either wrapped or by using the `for` property with `id`.

File: packages/eds-core-react/src/components/Input/Input.test.tsx [hunk 1]
```diff
@@ -26,6 +27,20 @@ describe('Input', () => {
     const { asFragment } = render(<Input value="input value" readOnly />)
     expect(asFragment()).toMatchSnapshot()
   })
+  it('Should pass a11y test when using label', async () => {
+    const label = 'Input'
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Input/Input.test.tsx:31]:
```suggestion
```

File: packages/eds-core-react/src/components/Input/Input.test.tsx [hunk 2]
```diff
@@ -26,6 +27,20 @@ describe('Input', () => {
...
+    const { container } = render(
+      <label>
+        {label}
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Input/Input.test.tsx:34]:
```suggestion
        Label text
```

**Review by mimarz — CHANGES_REQUESTED** [review]
Suggest just typing the label text in the label, noe need for variable interpolation if its not needed.

**Pull request merged**